sdk/servicebus/azure-servicebus/azure/servicebus/management/aio/_management_client_async.py

```diff
@@ -0,0 +1,259 @@
+
+
+if TYPE_CHECKING:
+    from azure.core.credentials_async import AsyncTokenCredential  # pylint:disable=ungrouped-imports
+
+
+class ServiceBusManagementClient:
+    """Use this client to create, update, list, and delete resources of a ServiceBus namespace
+
+    :param str fully_qualified_namespace:
+    :param credential: To authenticate to manage the entities of the ServiceBus namespace.
+    :type credential: Union[TokenCredential, ServiceBusSharedKeyCredential]
+    """
+
+    def __init__(self, fully_qualified_namespace, credential, **kwargs):
+        # type: (str, Union[AsyncTokenCredential, ServiceBusSharedKeyCredential], Dict[str, Any]) -> None
+
+        self.fully_qualified_namespace = fully_qualified_namespace
+        self._credential = credential
+        self._endpoint = "https://" + fully_qualified_namespace
+        self._config = ServiceBusManagementClientConfiguration(self._endpoint, **kwargs)
+        self._pipeline = self._build_pipeline()
+        self._impl = ServiceBusManagementClientImpl(endpoint=fully_qualified_namespace, pipeline=self._pipeline)
+
+    def _build_pipeline(self, **kwargs):  # pylint: disable=no-self-use
+        transport = kwargs.get('transport')
+        policies = kwargs.get('policies')
+        credential_policy = \
+            AsyncServiceBusSharedKeyCredentialPolicy(self._endpoint, self._credential, "Authorization") \
+            if isinstance(self._credential, ServiceBusSharedKeyCredential) \
+            else AsyncBearerTokenCredentialPolicy(self._credential, JWT_TOKEN_SCOPE)
+        if policies is None:  # [] is a valid policy list
+            policies = [
+                RequestIdPolicy(**kwargs),
+                self._config.headers_policy,
+                self._config.user_agent_policy,
+                self._config.proxy_policy,
+                ContentDecodePolicy(**kwargs),
+                self._config.redirect_policy,
+                self._config.retry_policy,
+                credential_policy,
+                self._config.logging_policy,
+                DistributedTracingPolicy(**kwargs),
+                HttpLoggingPolicy(**kwargs),
+            ]
+        if not transport:
+            transport = AioHttpTransport(**kwargs)
+        return AsyncPipeline(transport, policies)
+
+    @classmethod
+    def from_connection_string(cls, connection_string, **kwargs):
+        # type: (str, Any) -> ServiceBusManagementClient
+        """Create a client from connection string
+
+        :param str connection_string: The connection string of the Service Bus Namespace
+        """
+        endpoint, shared_access_key_name, shared_access_key, _ = parse_conn_str(connection_string)
+        if "//" in endpoint:
+            endpoint = endpoint[endpoint.index("//")+2:]
+        return cls(endpoint, ServiceBusSharedKeyCredential(shared_access_key_name, shared_access_key), **kwargs)
+
+    async def _get_queue_object(self, queue_name, clazz):
```

Comment:
clazz?

Reply:
apparently this is the java-ism of "class_" :P  I change elsewhere and briefly discussed this with Yijun, but yeah, fully agreed that we should be pythonic.

Reply:
yes, clazz is widely used in Java for "class_". updated to "class_"

Reply:
I also removed "class_". Will add this back when we have Topic, subscriptions, rules.